.gitmodules

```diff
@@ -1,3 +1,3 @@
-[submodule "/Users/james/devel/ExcessivelySafeCall/lib/ds-test"]
-	path = /Users/james/devel/ExcessivelySafeCall/lib/ds-test
+[submodule "lib/ds-test"]
```

Comment:
Had to update this to get submodule cloning to work

Reply:
Got updated to latest commit too, can revert that if you're opinionated about it